packages/server/modules/core/repositories/scheduledTasks.ts

```diff
@@ -0,0 +1,28 @@
+import { ScheduledTasks } from '@/modules/core/dbSchema'
+import { ScheduledTaskRecord } from '@/modules/core/helpers/types'
+
+export async function getLastScheduledTask(
+  taskName: string | null = null
+): Promise<ScheduledTaskRecord | null> {
+  let query = ScheduledTasks.knex<ScheduledTaskRecord>().orderBy(
+    ScheduledTasks.col.updatedAt,
+    'desc'
+  )
+  if (taskName) query = query.where({ taskName })
+  const task = await query.first()
+  return task ?? null
+}
+
+export async function saveScheduledTask(
+  scheduledTask: ScheduledTaskRecord
+): Promise<ScheduledTaskRecord> {
+  const [record] = await ScheduledTasks.knex<ScheduledTaskRecord>()
+    .insert(scheduledTask)
+    .onConflict(['createdAt', 'taskName'])
+    // this puts the full table name + col name into string, which doesn't work here
```

Comment:
you can use `ScheduledTasks.withoutTablePrefix` here

Reply:
nice, it was giving me a headache without that